src/Mutation/UpdateSettings.php

```diff
@@ -100,16 +100,7 @@ public static function get_output_fields() {
 		if ( ! empty( $allowed_setting_groups ) && is_array( $allowed_setting_groups ) ) {
 			foreach ( $allowed_setting_groups as $group => $setting_type ) {
 
-				$replaced_group = preg_replace( '[^a-zA-Z0-9 -]', ' ', $group );
```

Comment:
Does it matter that this preg_replace substitutes a space and the one below in [TypeRegistry an underscore](src/Registry/TypeRegistry.php)?